src/NCalc/Expression.cs

```diff
@@ -78,117 +79,85 @@ public static bool CacheEnabled
                 if (!CacheEnabled)
                 {
                     // Clears cache
-                    _compiledExpressions = new ConcurrentDictionary<string, WeakReference>();
+                    _compiledExpressions.Clear();
                 }
             }
         }
 
         /// <summary>
         /// Removed unused entries from cached compiled expression
         /// </summary>
-        private static void CleanCache()
+        private static void ClearCache()

```

Comment:
should we keep the old name? we are not really clearing it - clean seems more appropriate for selectively removing entries